test/src/087-xattrs/main

```diff
@@ -0,0 +1,21 @@
+cvmfs_test_name="Test magic extended attributes"
+cvmfs_test_suites="quick"
+
+cvmfs_run_test() {
+  logfile=$1
+
+  cvmfs_mount grid.cern.ch || return 1
+  local attributes_err = $(list_xattrs /cvmfs/grid.cern.ch 2>&1 1>/dev/null)
```

Comment:
Not sure what is being tested here.  Isn't the output always empty by definition?

Reply:
Fuse somehow tests the actual values of the attributes even when just listing them. When I broke the repo_counters attribute, list_xattrs actually outputted an error.